### PR TITLE
Merge test-branch with master so latest commits from the past 7 days are synced with original react repository

### PR DESCRIPTION
**Pull request by leannabbtest**
Here's the default description that was set up for this repo that I'm retaining, and adding some images.

IMG_7502

IMG_7502

IMG_7502

IMG_7502

IMG_7502

IMG_7502

**Before submitting a pull request,** please make sure the following is done:

1. Fork [the repository](https://github.com/facebook/react) and create your branch from `master`.
2. Run `yarn` in the repository root.
3. If you've fixed a bug or added code that should be tested, add tests!
4. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch TestName` is helpful in development.
5. Run `yarn test-prod` to test in the production environment. It supports the same options as `yarn test`.
6. If you need a debugger, run `yarn debug-test --watch TestName`, open `chrome://inspect`, and press "Inspect".
7. Format your code with [prettier](https://github.com/prettier/prettier) (`yarn prettier`).
8. Make sure your code lints (`yarn lint`). Tip: `yarn linc` to only check changed files.
9. Run the [Flow](https://flowtype.org/) typechecks (`yarn flow`).
10. If you haven't already, complete the CLA.

**Learn more about contributing:** https://reactjs.org/docs/how-to-contribute.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,6 +104,7 @@
     "debug-test": "cross-env NODE_ENV=development node --inspect-brk node_modules/.bin/jest --config ./scripts/jest/config.source.js --runInBand",
     "test": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.source.js",
     "test-persistent": "cross-env NODE_ENV=development jest --config ./scripts/jest/config.source-persistent.js",
+    "debug-test-persistent": "cross-env NODE_ENV=development node --inspect-brk node_modules/.bin/jest --config ./scripts/jest/config.source-persistent.js --runInBand",
```
Comment (Blooooop) [inline package.json]: Inserting a code suggestion 

```suggestion
    "test-persistent": "cross-env NODE_ENV=development node --inspect-brk node_modules/.bin/jest --config ./scripts/jest/config.source-persistent.js --runInBand",
```

File: fixtures/dom/src/index.test.js [hunk 2]
```diff
@@ -25,153 +24,210 @@ function App(props) {
   return 'hello world';
```
Comment (Blooooop) [inline fixtures/dom/src/index.test.js]:
```suggestion
  return 'hello nerds';
```

Comment (Blooooop) [inline fixtures/dom/src/index.test.js]: 'hello world' is a bit tired and overused. What about 'hello nerds'? I think the term 'nerd' now has a much more positive connotation, and I personally think it's endearing. Just trying to come up with a random string of text to test out this comment feature!  

```suggestion
  return 'hello nerds';
```

File: package.json (hunk 1 above)
Comment (Blooooop) [inline package.json:107]: Ignore this comment and just resolve the conversation

**Review by Blooooop — COMMENTED** [review]
Submitting a "Comment" review (submit general feedback without explicit approval).

**Comment by Blooooop** [comment]
Leaving a PR-wide comment

**Comment by Blooooop** [comment]
Leaving a PR-wide comment (let's try this again..)

File: fixtures/dom/src/index.test.js (hunk 2 above)
Comment (leannabbtest) [reply fixtures/dom/src/index.test.js]: Thanks for the feedback! I've committed your suggestion.

File: package.json (hunk 1 above)
Comment (leannabbtest) [reply package.json:107]: Will do!

**Review by Blooooop — APPROVED** [review]
Everything looks great!

**Pull request merged**